sql/2019/04_Media/04_01a.sql

```diff
@@ -0,0 +1,105 @@
+#standardSQL
+/*
+04_01a
+
+1.22 TB query
+*Lighthouse Lazyloading, resp images, image quality, webp image score score in 1000 percentile points	
+
+this query takes lighthouse data on image performance: 
+    lazyloading (offscreen), 
+    responsiveness, 
+    quality (optimized),
+    format (webp),
+    animated GIF
```

Comment:
This query doesn't include the gif score. Could you update the comments to match what's included in the query? For example, it also talks about 50 quantiles below.